robolectric/src/test/java/org/robolectric/shadows/ShadowNotificationManagerTest.java

```diff
@@ -93,4 +97,32 @@ public void testCancelAll() throws Exception {
     assertNull(shadowOf(notificationManager).getNotification(null, 1));
     assertNull(shadowOf(notificationManager).getNotification(null, 31));
   }
+
+  @Test
+  public void testGetActiveNotifications() throws Exception {
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
+      // This API was added in M.
+      return;
+    }
+    notificationManager.notify(1, notification1);
+    notificationManager.notify(31, notification2);
+
+    StatusBarNotification[] statusBarNotifications =
+        shadowOf(notificationManager).getActiveNotifications();
+    assertEquals(2, statusBarNotifications.length);
+    boolean hasNotification1 = false;
+    boolean hasNotification2 = false;
+    for (StatusBarNotification notification : statusBarNotifications) {
+      if (notification.getId() == 1) {
+        hasNotification1 = true;
+        assertEquals(notification1, notification.getNotification());
```

Comment:
Done - I had things this way to be consistent with the other tests in this class, but whatever you prefer is fine with me. Note that I reversed the order since it seems more canonical to have the object being tested come first. (Which makes sense if you want to change the condition you're checking it against).